src/main/resources/prevac-config.json

```diff
@@ -7,5 +7,6 @@
     "Max Seventh Long Term Follow Up Visits", "Max Third Vaccination Visits", "Max First Post Third Vaccination Visits",
     "Max Second Post Third Vaccination Visits", "Max Third Post Third Vaccination Visits",
     "Max Fourth Post Third Vaccination Visits", "Max Fifth Post Third Vaccination Visits"],
+  "boosterRelatedVisits" :[],
```

Comment:
Look into documentation and add visits that are related to booster (Boost Vaccination First Follow-up Visit)